models/setup.py

```diff
@@ -0,0 +1,39 @@
+import pip
```

Comment:
Is it documented how to run integration tests locally?

Reply:
i added it to CONTRIBUTING.md